components/eam/bld/build-namelist

```diff
@@ -138,6 +138,23 @@ my $commandline = "$cfgdir/build-namelist @ARGV";
 
 # Process command-line options.
 
+if ( -e "../../user_nl_cam") {
```

Comment:
Do we store path to the case directory somewhere in this script? It will be better to use that instead of a relative path. If not, this should also be okay.

Reply:
Thanks for the quick check, @singhbalwinder . build-namelist always runs from Buildconf/eamconf (until it is ever changed), so I feel safe just to use the relative path. Plus, we expect to remove it in probably not too distant future after we all get use to it.

Reply:
To answer your question, Balwinder.  build-namelist ran with several command-line options, including the following which can be parse to get the case directory
`-infile /global/cfs/cdirs/e3sm/wlin/E3SM_testings/test_user_nl_cam/case_scripts/Buildconf/eamconf/cesm_namelist`

After a check at the point of build-namelist is invoked, I am surprised to see no case specific ENV variables available to it, other than 'PWD', which is the fullpath to Buildconf/eamconf.

Reply:
Thanks for looking into it. I think it is fine as it is as (like you said), it is going to be temporary.